### PR TITLE
[OBSDEF-10105] Make iam and federation pods Guaranteed

### PR DESCRIPTION
**Pull request by voa808**
## Purpose
[OBSDEF-10105] Make iam and federation pods Guaranteed
Master PR: https://github.com/EMCECS/charts/pull/923
_List the changes for this PR_

## PR checklist
- [ ] make test passed
- [x] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed
- [x] Passed - https://asd-ecs-jenkins.isus.emc.com/jenkins/view/ECS-Flex/job/charts-custom-ci/

## Testing
https://asd-ecs-jenkins.isus.emc.com/view/ECS-Flex/job/charts-custom-ci/319/

_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by victor-polyansky — APPROVED** [review]

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

File: objectscale-iam/values.yaml [hunk 1]
```diff
@@ -115,6 +117,8 @@ atlas:
   # Atlas pods limits and requests memory setting
   resources:
     limits:
-      memory: 1000Mi
+      memory: 400Mi
+      cpu: 400m
```
[the remark this answers is not in the document]
Comment (voa808) [reply objectscale-iam/values.yaml:121]: It means milicpu. 
Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu

**Comment by voa808** [comment]
This fix is dependent on monitoring PR https://eos2git.cec.lab.emc.com/ECS/monitoring/pull/667  which is merged to master. As confirmed with Victor, there's no branching in monitoring, beta3 charts should be updated and our fix for Guaranteed pods should work. The tests were done with the provided custom builds by monitoring. 
Reference: https://delloneisg.slack.com/archives/CM7QFA3EV/p1630355378009000

**Review by lipak2345 — APPROVED** [review]

File: objectscale-iam/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eddiepavkovic) [reply objectscale-iam/values.yaml:121]: Thanks I meant it should be added as a comment in the code

**Review by eddiepavkovic — APPROVED** [review]
G2g.   Please check off the charts CI custom passed

**Comment by voa808** [comment]
> G2g. Please check off the charts CI custom passed

https://asd-ecs-jenkins.isus.emc.com/view/ECS-Flex/job/charts-custom-ci/319/ passed.

**Pull request merged**